fineract-provider/src/main/java/org/apache/fineract/portfolio/self/shareaccounts/api/SelfShareAccountsApiResource.java

```diff
@@ -139,7 +152,11 @@ public String createAccount(final String apiRequestBodyAsJson) {
 	@Path("{accountId}")
 	@Consumes({ MediaType.APPLICATION_JSON })
 	@Produces({ MediaType.APPLICATION_JSON })
-	public String retrieveShareAccount(@PathParam("accountId") final Long accountId, @Context final UriInfo uriInfo) {
+	/*
+    @ApiOperation(value = "Retrieve a share application/account", httpMethod = "GET", notes = "\n" +
+            "\n" + "\n" + "Example Requests:\n" + "\n" + "self/shareaccounts/12\n")
+    @ApiResponses({@ApiResponse(code = 200, message = "OK", responseContainer = "List", response  SelfShareAccountsApiResourceSwagger.GetShareAccountResponse.class)})*/
```

Comment:
Why these annotations is commented??

Reply:
@Sanyam96 FYI I've fixed this one in my swagger-squash branch which I'll merge ASAP as much as possible. @kangbreder the problem is that SelfShareAccountsApiResourceSwagger.GetShareAccountResponse seems to be missing- would you like to raise a small new PR just adding that and fixing this?